server/src/main/java/com/coincoinche/websockets/controllers/GameController.java

```diff
@@ -130,21 +132,36 @@ public void move(
       // should never happen
       logger.error("Inconsistent error: legal move is illegal: {}", move);
       e.printStackTrace();
+    } catch (UserNotFoundException e) {
+      e.printStackTrace();
```

Comment:
Can we add an error log along the lines of `failed to terminate game`? so that we know right away what is going wrong